packages/lesmis-server/src/connections/SessionHandlingApiConnection.ts

```diff
@@ -0,0 +1,43 @@
+/* eslint-disable max-classes-per-file */
```

Comment:
Personally I'd love for us to just disable this rule entirely? I think creating one class per file is just standard practice and I don't really need a rule to remind me of this... and in the scenario that we choose to break this rule we know what we're doing and why... How do you feel about it?

Reply:
Yeah I'm with you there! @kael89 how about you?

Reply:
I'm going to leave this for now and raise it at dev chat

Reply:
I agree with you, let's just disable it!